### PR TITLE
gendesk: update to 1.0.9.

### PR DESCRIPTION
**Pull request by benalb**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/gendesk/template [hunk 1]
```diff
@@ -12,7 +12,7 @@ license="MIT"
 homepage="http://roboticoverlords.org/gendesk/"
 distfiles="https://github.com/xyproto/gendesk/archive/${version}.tar.gz
  http://roboticoverlords.org/images/default.png"
```
Comment (paper42) [inline srcpkgs/gendesk/template:14]:
```suggestion
homepage="https://roboticoverlords.org/gendesk/"
distfiles="https://github.com/xyproto/gendesk/archive/${version}.tar.gz
 https://roboticoverlords.org/images/default.png"
```

**Review by paper42 — COMMENTED** [review]
We might be able to remove these two lines:
```
go_mod_mode=default
hostmakedepends="git"
```

File: srcpkgs/gendesk/template (hunk 1 above)
Comment (benalb) [reply srcpkgs/gendesk/template]: Done.

**Comment by paper42** [comment]
squash the commits, otherwise lgtm :+1:

**Comment by benalb** [comment]
Sorry for the extra work for the builders, In my defense, I got the booster dose and it hit me hard.

**Comment by ericonr** [comment]
No worries, Go build are plenty quick ;)

**Pull request merged**